src/templates_front/build.sh

```diff
@@ -3,6 +3,12 @@
 # prefixes
 cat prefix_front.c   > trexio.c
 cat prefix_front.h   > trexio.h
+# grep some usefull constants from the config.h
+echo "" >> trexio.h
+grep "_VERSION" ../../include/config.h >> trexio.h
+grep "_USER_NAME" ../../include/config.h >> trexio.h
+echo "" >> trexio.h
+
```

Comment:
Instead of doing a `grep`, we could probably replace `config.h` by `trexio.h` in the `configure.ac` file in order to have only one file, no?